src/fmu/sumo/utilities/sim2sumo.py

```diff
@@ -0,0 +1,423 @@
+
+    Returns:
+        str: derived name
+    """
+    datafile_path_posix = Path(datafile_path)
+    base_name = datafile_path_posix.name.replace(
+        datafile_path_posix.suffix, ""
+    )
+    while base_name[-1].isdigit() or base_name.endswith("-"):
+        base_name = base_name[:-1]
+    return base_name
+
+
+def convert_to_arrow(frame):
+    """Convert pd.DataFrame to arrow
+
+    Args:
+        frame (pd.DataFrame): the frame to convert
+
+    Returns:
+        pa.Table: the converted dataframe
+    """
+    table = Table.from_pandas(frame)
+    return table
+
+
+def get_dataframe(
+    datafile_path: str, submod: str, print_help=False, **kwargs
+) -> pd.DataFrame:
+    """Fetch dataframe from simulator results
+
+    Args:
+        datafile_path (str): the path to the simulator datafile
+        submod (str): the name of the submodule to extract with
+        kwargs (dict): other options
+
+    Returns:
+        pd.DataFrame: the extracted data
+    """
+    logger = logging.getLogger(__file__ + ".get_dataframe")
+    extract_df = SUBMOD_DICT[submod]["extract"]
+    arrow = kwargs.get("arrow", False)
+    frame = None
+    if print_help:
+        print(SUBMOD_DICT[submod]["doc"])
+    else:
+        right_kwargs = {
+            key: value
+            for key, value in kwargs.items()
+            if key in SUBMOD_DICT[submod]
+        }
+        logger.debug("Exporting with arguments %s", right_kwargs)
+        try:
+            frame = extract_df(sim2df.EclFiles(datafile_path), **right_kwargs)
+            if arrow:
+                try:
+                    frame = kwargs["arrow_convertor"](frame)
+                except KeyError:
+                    logger.debug("No arrow convertor defined for %s", submod)
+                    frame = convert_to_arrow(frame)
+
+        except TypeError:
+            logger.warning(
+                "Couldn't produce results!! Most likely something with you call"
+            )
+        except FileNotFoundError:
+            logger.warning(
+                "Cannot produce results!!, most likely no results present"
+            )
+    return frame
+
+
+def check_options(submod, key_args):
+    """Check keyword args against possible optiopns
+
+    Args:
+        submod (str): the submodule to use from sim2df
+        key_args (dict): the keyword arguments supplied
+
+    Raises:
+        KeyError: if key(s) in key_args not available for submodule
+    """
+    unknowns = [
+        key
+        for key in key_args.keys()
+        if key not in SUBMOD_DICT[submod]["options"]
+    ]
+    if len(unknowns) > 0:
+        raise KeyError(
+            (
+                f"these unknown options are included in call {unknowns}\n"
+                + f"please read the docs: \n {SUBMOD_DICT[submod]['doc']}"
+            )
+        )
+
+
+def export_csv(
+    datafile_path: str,
+    submod: str,
+    global_variables_file="fmuconfig/output/global_variables.yml",
```

Comment:
This default will only work on some assets - but there isn't an argument in the FORWARD_MODEL definition to put in an alternate. How do I provide a different path to the code?

Reply:
There is now.. See updated code